core/src/main/resources/hudson/PluginManager/installed_de.properties

```diff
@@ -21,8 +21,8 @@
 # THE SOFTWARE.
 
 No\ plugins\ installed.=Keine Plugins installiert.
-Changes\ will\ take\ effect\ when\ you\ restart\ Jenkins=�nderungen treten nach einem Neustart von Jenkins in Kraft.
-Uncheck\ to\ disable\ the\ plugin=Zum Deaktivieren des Plugins Markierung l�schen
+Changes\ will\ take\ effect\ when\ you\ restart\ Jenkins=\u00C4nderungen treten nach einem Neustart von Jenkins in Kraft.
```

Comment:
> I propose to use this PR to consequently switch to German umlauts in German localization.

I never change them; my IDE does it for me. As I see no reason to ever edit these file directly (it's too error prone anyway), I don't care.